Tests/DependencyInjection/SncRedisExtensionTest.php

```diff
@@ -308,6 +308,41 @@ public function testClientReplicationOption()
         $this->assertEquals(array('snc_redis.default' => array(array('alias' => 'default'))), $container->findTaggedServiceIds('snc_redis.client'));
     }
 
+    /**
+     * Test valid config of the serialization option
+     */
+    public function testClientSerializationOption()
+    {
+         $extension = new SncRedisExtension();
+         $config = $this->parseYaml($this->getSerializationYamlConfig());
+         $extension->load(array($config), $container = $this->getContainer());
+         $options = $container->getDefinition('snc_redis.client.default_options')->getArgument(0);
+         $parameters = $container->getDefinition('snc_redis.default')->getArgument(0);
+         $masterParameters = $container->getDefinition((string) $parameters[0])->getArgument(0);
+         $this->assertSame($options['serialization'], $masterParameters['serialization']);
+    }
+
+    /**
+     * Test validity of serialization type
+     */
+    public function testLoadSerializationType()
+    {
+        $extension = new SncRedisExtension();
+        $config = $this->parseYaml($this->getSerializationYamlConfig());
+        $extension->load(array($config), $container = $this->getContainer());
+        $options = $container->getDefinition('snc_redis.client.default_options')->getArgument(0);
+        $serializationType = $extension->loadSerializationType($options['serialization']);
+        $this->assertTrue(is_integer($serializationType));
+
+        if (defined('HHVM_VERSION')) {
```

Comment:
Same here - it's a new feature so don't need to support HHVM for BC.

Reply:
will remove now